python/generateMenuFromJSON.py

```diff
@@ -44,13 +48,21 @@ def generateLogger(evtMode, outdir, dictLog, logName, dictStreams, isOfflineBuil
         list_of_logger_streams.append(streamName)
         #
         if doIt == True:
-            loggerConfig.write('   {}:'.format(streamName)+' { \n')
-            loggerConfig.write('      module_type: RootDAQOutput \n')
-            loggerConfig.write('      SelectEvents : {}\n'.format(json.dumps(dictStreams[k])))
-            loggerConfig.write('      maxSubRuns   : 1\n')
-            loggerConfig.write('      fileName     : "{}.art\"\n'.format(k))
-            loggerConfig.write('   }\n\n')
+            loggerConfig.write('         {}:'.format(streamName)+' { \n')
+            loggerConfig.write('            module_type: RootDAQOutput \n')
+            loggerConfig.write('            SelectEvents : {}\n'.format(json.dumps(dictStreams[k])))
+            loggerConfig.write('            maxSubRuns   : 1\n')
+            loggerConfig.write('            fileName     : "{}_%r_%#.art\"\n'.format(k))
+            loggerConfig.write('         }\n\n')
+    if doIt == True:
```

Comment:
I don't know enough python to understand why you split the following logic into 3 stanzas.   It looks odd but not broken.